extensions/interactions/NumberWithUnits/directives/NumberWithUnits.js

```diff
@@ -135,9 +136,29 @@ oppia.directive('oppiaShortResponseNumberWithUnits', [
   }
 ]);
 
-// Rule evaluation for number with units (will be implemented in M1.3).
+// Rule evaluation for number with units.
```

Comment:
Rules service for number with units interaction.

Reply:
Done.